antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,286 @@
+                                {% if choice is sameas true %}
+                                    {% set choice = 'yes' %}
+                                {% elif choice is sameas false %}
+                                    {% set choice = 'no' %}
+                                {% endif %}
+                                {% if (value['default'] is not list and value['default'] == choice) or (value['default'] is list and choice in value['default']) %}
+                                    <li><div style="color: blue"><b>@{ choice | escape }@</b>&nbsp;&larr;</div></li>
+                                {% else %}
+                                    <li>@{ choice | escape }@</li>
+                                {% endif %}
+                            {% endfor %}
+                        </ul>
+                    {% endif %}
+                    {# Show default value, when multiple choice or no choices #}
+                    {% if value['default'] is not none and value['default'] not in value['choices'] %}
+                        <b>Default:</b><br/><div style="color: blue">@{ value['default'] | tojson | escape }@</div>
+                    {% endif %}
+                </td>
+                {# description #}
+                <td>
+                    {% for desc in value['description'] %}
+                        <div>@{ desc | replace('\n', '\n    ') | html_ify }@</div>
+                    {% endfor %}
+                    {% if value['aliases'] %}
+                        <div style="font-size: small; color: darkgreen"><br/>aliases: @{ value['aliases']|join(', ') }@</div>
+                    {% endif %}
+                </td>
+            </tr>
+            {% if value['options'] %}
+                @{ loop(value['options']|dictsort) }@
+            {% endif %}
+        {% endfor %}
+    </table>
+    <br/>
+{%   endif %}
+
+.. Notes
+
+{%   if ep_doc['notes'] -%}
+Notes
+^^^^^
+
+.. note::
+{%     for note in ep_doc['notes'] %}
+   - @{ note | rst_ify | indent(width=5) }@
+{%     endfor %}
+{%   endif %}
+
+.. Seealso
+
+{%   if ep_doc['seealso'] -%}
+See Also
+^^^^^^^^
+
+.. seealso::
+
+{%     for item in ep_doc['seealso'] %}
+{%       if item.module is defined and item.description %}
+   @{ ('M(' + item['module'] + ')') | rst_ify }@
+       @{ item['description'] | rst_ify }@
+{%       elif item.module is defined %}
+   @{ ('M(' + item['module'] + ')') | rst_ify }@
+      The official documentation on the **@{ item['module'] }@** module.
+{%       elif item.name is defined and item.link is defined and item.description %}
+   `@{ item['name'] }@ <@{ item['link'] }@>`_
+       @{ item['description'] | rst_ify }@
+{%       elif item.ref is defined and item.description %}
+   :ref:`@{ item['ref'] }@`
+       @{ item['description'] | rst_ify }@
+{%       endif %}
+{%     endfor %}
+{%   endif %}
+
+{%   if ep_doc['author'] -%}
+Authors
+^^^^^^^
+
+{%     for author_name in ep_doc['author'] %}
+- @{ author_name }@
```

Comment:
Do we. have this metadata?

Reply:
This section is only added when `author` is present. Some roles do provide this.